projects/plugins/jetpack/sal/class.json-api-date.php

```diff
@@ -47,7 +47,7 @@ public static function format_date( $date_gmt, $date = null ) {
 						$offset = $timezone->getOffset( $date_time );
 					}
 				} else {
-					$offset = 3600 * get_option( 'gmt_offset' );
+					$offset = 3600 * floatval( get_option( 'gmt_offset' ) );
```

Comment:
Any reason we are not casting here? eg 
```suggestion
					$offset = 3600 * (float)get_option( 'gmt_offset', 0 );
```

Reply:
Not really, the default option should not be needed since casting false should still return 0.0

Reply:
Added it in [82ca32c](https://github.com/Automattic/jetpack/pull/43123/commits/82ca32c7b534be0fd0cd31a7923211dbc4b36e97)